edgar-app/src/components/dashboardPages/home/ChatCard.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, HStack, Icon, Text, VStack, useDisclosure } from '@chakra-ui/react';
+
+import CircleRightArrowIcon from 'assets/icons/Arrow/Circle/CircleRightArrowIcon';
+
+import CircleZeroIcon from 'assets/icons/CircleNumber/CircleZeroIcon';
+import CircleOneIcon from 'assets/icons/CircleNumber/CircleOneIcon';
+import CircleTwoIcon from 'assets/icons/CircleNumber/CircleTwoIcon';
+import CircleThreeIcon from 'assets/icons/CircleNumber/CircleThreeIcon';
+import CircleFourIcon from 'assets/icons/CircleNumber/CircleFourIcon';
+import CircleFiveIcon from 'assets/icons/CircleNumber/CircleFiveIcon';
+import CircleSixIcon from 'assets/icons/CircleNumber/CircleSixIcon';
+import CircleSevenIcon from 'assets/icons/CircleNumber/CircleSevenIcon';
+import CircleEightIcon from 'assets/icons/CircleNumber/CircleEightIcon';
+import CircleNineIcon from 'assets/icons/CircleNumber/CircleNineIcon';
+import CirclePlusIcon from 'assets/icons/CircleNumber/CirclePlusIcon';
+
+import { MessageType } from 'types/dashboard/home/MessageType';
+
+const ChatCard = ({ message }: { message: MessageType }): JSX.Element => {
+	const { isOpen: isHover, onOpen: onHoverOpen, onClose: onHoverClose } = useDisclosure();
+	const icons = [
+		{ icon: CircleZeroIcon },
+		{ icon: CircleOneIcon },
+		{ icon: CircleTwoIcon },
+		{ icon: CircleThreeIcon },
+		{ icon: CircleFourIcon },
+		{ icon: CircleFiveIcon },
+		{ icon: CircleSixIcon },
+		{ icon: CircleSevenIcon },
+		{ icon: CircleEightIcon },
+		{ icon: CircleNineIcon },
+	];
+
+	return (
+		<HStack
+			bg="blue.100"
+			w="100%"
+			borderRadius="16px"
+			border="2px"
+			borderColor="blue.200"
+			spacing="8px"
+			onMouseEnter={onHoverOpen}
+			onMouseLeave={onHoverClose}
+		>
+			<HStack w="100%" bg="white" borderRadius="16px" p="8px 16px" justify="space-between">
+				<HStack spacing="8px">
+					<Box borderRadius="50%" bg="green.300" w="28px" h="28px" />
+					<VStack align="start" spacing="0px">
+						<HStack spacing="4px">
+							<Text size="boldMd">{message.sender}</Text>
+							<Text fontStyle="italic" fontSize="12" color="grey.500" fontWeight="500" lineHeight="18px">
+								{/* Insérer - Médecin généraliste */}
```

Comment:
You need to add this text, you can have access to the id of the primary doctor thanks to the medical info